### ISSUE TITLE
Cannot find module './gulpfile.coffee'

### ISSUE THREAD

**Issue by alvarocantador** [CinKon/generator-uikit-webapp#4]
`gulp` command in the root project folder, returns this error:

```
alvaro@machine:~/Workspace/snippets/tdfront$ gulp
module.js:341
    throw err;
    ^

Error: Cannot find module './gulpfile.coffee'
  at Function.Module._resolveFilename (module.js:339:15)
  at Function.Module._load (module.js:290:25)
  at Module.require (module.js:367:17)

```

Node version: 5.10.1
Npm: 3.8.6
Yeoman: 1.7.0
SO: Debian 8.4

**Comment by CinKon** [comment]
Looks like the `gulpfile.coffee` is missing in your folder. 
Can you check that?
Did Yeoman throw any error-messages while installation?

**Comment by alvarocantador** [comment]
Project root:
```
alvaro@machine:~/Workspace/project$ ls -la
total 176
drwxr-xr-x   8 alvaro alvaro  4096 Abr 28 14:24 .
drwxr-xr-x   7 alvaro alvaro  4096 Abr 28 14:16 ..
-rw-r--r--   1 alvaro alvaro  1272 Abr 28 14:18 404.html
-rw-r--r--   1 alvaro alvaro  3959 Abr 28 14:18 apple-touch-icon.png
drwxr-xr-x   6 alvaro alvaro  4096 Abr 28 14:23 bower
-rw-r--r--   1 alvaro alvaro   448 Abr 28 14:18 bower.json
-rw-r--r--   1 alvaro alvaro    27 Abr 28 14:18 .bowerrc
-rw-r--r--   1 alvaro alvaro   416 Abr 28 14:18 browserconfig.xml
-rw-r--r--   1 alvaro alvaro   611 Abr 28 14:18 crossdomain.xml
drwxr-xr-x   2 alvaro alvaro  4096 Abr 28 14:18 css
-rw-r--r--   1 alvaro alvaro   191 Abr 28 14:18 .editorconfig
-rw-r--r--   1 alvaro alvaro   766 Abr 28 14:18 favicon.ico
-rw-r--r--   1 alvaro alvaro  6909 Abr 28 14:18 Gulpfile.coffee
-rw-r--r--   1 alvaro alvaro   166 Abr 28 14:18 Gulpfile.js
-rw-r--r--   1 alvaro alvaro 39014 Abr 28 14:18 .htaccess
-rw-r--r--   1 alvaro alvaro   229 Abr 28 14:18 humans.txt
drwxr-xr-x   2 alvaro alvaro  4096 Abr 28 14:18 img
-rw-r--r--   1 alvaro alvaro 22023 Abr 28 14:18 index.html
drwxr-xr-x   2 alvaro alvaro  4096 Abr 28 14:18 js
-rw-r--r--   1 alvaro alvaro   293 Abr 28 14:18 .jshintrc
-rw-r--r--   1 alvaro alvaro  1356 Abr 28 14:18 modernizr-config.json
drwxr-xr-x 694 alvaro alvaro 20480 Abr 28 14:24 node_modules
-rw-r--r--   1 alvaro alvaro   754 Abr 28 14:24 package.json
-rw-r--r--   1 alvaro alvaro    78 Abr 28 14:18 robots.txt
drwxr-xr-x   5 alvaro alvaro  4096 Abr 28 14:18 src
```

No errors during instalation.

**Comment by CinKon** [comment]
I think the Problem is the naming. Just rename Gulpfile.coffee to gulpfile.coffee (lowercase). Zhat should do it.
I'll fix it next Week in the repository

**Comment by alvarocantador** [comment]
Works fine! Thanks!

**Comment by alvarocantador** [comment]
I created a pull request to fix this #5
